versions/3.0.4.md

```diff
@@ -2711,7 +2711,7 @@ There are two ways to define the value of a discriminator for an inheriting inst
 
 ###### XML Modeling
 
-The [xml](#schema-xml) property allows extra definitions when translating the JSON definition to XML.
+The [xml](#schema-xml) field allows extra definitions when translating the JSON definition to XML.
```

Comment:
I think there is some ambiguity in our rules here. In later versions of the spec (not this one since it is based on Draft 5 which does not have vocabularies), the `xml` field/keyword/thingy is defined in a JSON Schema vocabulary. When that is the case, do we consider it part of JSON Schema and call it a "keyword" or an OpenAPI fixed field and call it a "field"? "discriminator" is probably another instance of this.

Reply:
Checked:
* https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00: only refers to "xml-names"
* https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00 does not mention xml at all

So it is an OpenAPI field.

Reply:
My question is more a philosophical one. If I write a JSON Schema vocabulary that adds keywords/fields/thingys, then are these called "keywords", following the JSON Schema terminology, or can I choose to call them "fields" or anything else I might choose to call them? Even if I _could_ call them _anything_, is that really a good thing to do?

Reply:
I personally define "terms" in a vocabulary, not "fields" or "keywords" 😎

Reply:
JSON Schema vocabularies define "extension keywords" according to @handrews